src/delete-activity-log.ts

```diff
@@ -116,16 +109,19 @@ export const batchDeleteActivityLog = async (
 };
 
 export const handler = async (event: SNSEvent): Promise<void> => {
-  const { DLQ_URL, TABLE_NAME } = process.env;
-
   await Promise.all(
     event.Records.map(async (record) => {
+      const { DLQ_URL, TABLE_NAME } = process.env;
+      if (!DLQ_URL) {
```

Comment:
We have this pattern a lot now. Do you think it'd be worth adding a new helper to get an environment variable? Something like this
```typescript
export const getEnvironmentVariable(name: string): string => {
  const value = process.env[name];
  if (!value) {
    throw new Error(`${name} environment variable is not set`);
  }
};
```

Reply:
I see in a later commit you moved the validation of these values into the try-catch block. I think it's probably correct to not do that - if we're missing a critical environment variable then we can fail early and like you said on Slack we don't need to catch every error. The Lambda service can pick up on the error state and handle the retries and DLQ for us.